third-party/gmp/Makefile

```diff
@@ -66,27 +81,32 @@ cleanall: FORCE
 clobber: FORCE
 	rm -rf build install $(GMP_SUBDIR)
 
-$(GMP_UNPACKED_DIR):
-	cd $(GMP_DIR) && tar --bzip2 -xf $(GMP_TARBALL)
-
 $(GMP_BUILD_SUBDIR):
 	mkdir -p $@
 
-gmp-config:
-	$(MAKE) $(GMP_MAKEFILE)
-
-$(GMP_MAKEFILE): $(GMP_BUILD_SUBDIR) $(GMP_UNPACKED_DIR)
-	cd $(GMP_BUILD_DIR) && $(GMP_SUBDIR)/configure CC='$(CC)' CFLAGS='$(CFLAGS) $(CHPL_GMP_CFLAGS)' CXX='$(CXX)' CXXFLAGS='$(CXXFLAGS) $(CHPL_GMP_CXXFLAGS)' $(CHPL_GMP_ABI_ARG) --prefix=$(GMP_INSTALL_DIR) $(CHPL_GMP_CFG_OPTIONS)
+$(GMP_UNPACKED_DIR):
+	cd $(GMP_DIR) && tar --bzip2 -xf $(GMP_TARBALL)
 
 gmp-build:
 	$(MAKE) $(GMP_H_FILE)
 
-$(GMP_H_FILE): $(GMP_MAKEFILE)
+$(GMP_H_FILE): $(GMP_BUILD_SUBDIR)
+ifeq (yes, $(GMP_SPECULATIVE_CRAY_X_PGI))
+	@echo 'Speculative build of gmp squashed due to PGI target compiler on Cray X*.'
+else
+ifeq (yes, $(GMP_CRAY_X_PGI))
+	@echo 'Forced gmp build with PGI on Cray X* fails some Chapel tests.'
+endif
+	@if [[ ! -e $(GMP_UNPACKED_DIR) ]]; then \
+	  $(MAKE) $(GMP_UNPACKED_DIR) ; \
+	fi
```

Comment:
Oh! Sorry, I misunderstood you. Yes, you're right; I was just saving the check and message, but the behavior may well be more clear without the test. I'll remove it.